client/cli/src/main.rs

```diff
@@ -83,7 +83,9 @@ enum CliOutput {
     },
 
     Info {
-        network: String,
+        federation_id: String,
```

Comment:
Can we impl `Serialize` for the federation id struct such that we don't have to manually turn it into a `String` here?